src/mobility/components/MobilityIconText.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {SvgProps} from 'react-native-svg';
+import {View} from 'react-native';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {ThemeText} from '@atb/components/text';
+import {useTheme} from '@atb/theme';
+
+type Props = {
+  svg?(props: SvgProps): JSX.Element;
+  text: string;
+};
+export const MobilityIconText = ({svg, text}: Props) => {
```

Comment:
If we want this as a separate component I believe it could be a local component inside `MobilityStat`, since that is the only place it's used. I also find the name a bit confusing. Is something like `MobilityStatWithIcon` more describing?

Reply:
Refactored, thanks!